modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/SiteResourceImpl.java

```diff
@@ -54,7 +54,8 @@
 				_groupService.getUserSitesGroups(
 					contextUser.getUserId(), pagination.getStartPosition(),
 					pagination.getEndPosition()),
-				this::_toSite));
+				this::_toSite),
+			pagination, _groupService.getUserSitesGroupsCount());
```

Comment:
@javierdearcos, changed so that we use the service instead of the localService and retrieve the count as you mentioned